cloud/azure/storage/storagesync/mode/heartbeat.pm

```diff
@@ -0,0 +1,143 @@
+#
+# Copyright 2021 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package cloud::azure::storage::storagesync::mode::heartbeat;
+
+use base qw(cloud::azure::custom::mode);
+
+use strict;
+use warnings;
+
+sub get_metrics_mapping {
+    my ($self, %options) = @_;
+
+    my $metrics_mapping = {
+        'storagesyncserverheartbeat' => {
+            'output' => 'Heartbeat',
+            'label'  => 'heartbeat',
+            'nlabel' => 'storage.storagesync.heartbeat',
```

Comment:
To align with our new naming convention, I would add the 'count' at the end of the metric
```suggestion
            'nlabel' => 'storagesync.heartbeat.count',
```